### ISSUE TITLE
WindowsError in bificial_radiance/main.py is platform specific - fails on linux/mac

### ISSUE THREAD

**Issue by mikofski** [NREL/bifacial_radiance#126]
here's the traceback:

```
(venv) mikm@PTOD702634:~$ PATH=/home/mikm/.local/opt/radiance/bin/:$PATH python
Python 2.7.12 (default, Nov 12 2018, 14:36:49)
[GCC 5.4.0 20160609] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> from bifacial_radiance import RadianceObj
>>> demo = RadianceObj(name = 'Testrun', path = 'myfolder')
path = myfolder
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/mikm/Projects/bifacial_radiance/bifacial_radiance/main.py", line 234, in __init__
    self._setPath(path)
  File "/home/mikm/Projects/bifacial_radiance/bifacial_radiance/main.py", line 248, in _setPath
    except WindowsError:
NameError: global name 'WindowsError' is not defined
>>> exit()
```

After some experimentation using `OSError` will on any platform (win32, linux, and mac osx) and any Python, but if using Python-3, then, `FileNotFoundError` also works.

**Comment by mikofski** [comment]
Even if you catch this error, there's still an error here:

https://github.com/NREL/bifacial_radiance/blob/d6bf98b3954782df15c97ab0b4d3370d704fa7dd/bifacial_radiance/main.py#L381-L382

because `os.path.join(self.path, material_path)` doesn't exist.

Also if `path` argument isn't a valid path, then several unexpected folders are created in whatever folder the interpreter was executed in.

https://github.com/NREL/bifacial_radiance/blob/d6bf98b3954782df15c97ab0b4d3370d704fa7dd/bifacial_radiance/main.py#L251-L258

Rather that create several folders where they were not expected, and then crash anyway, better to raise an exception right away. It's not necessary to catch the exception, to provide a custom message, but that's the way the code is currently written, so maybe better to discuss, or open another issue.

Also, IMO it's better to use logging instead of `print()` for debugging.

**Comment by jalderman9** [comment]
I'm running linux too and had the same issue.  I fixed it by adding a line to set material_path:

```        
        if platform.system() == "Linux":
            material_path = os.path.join(os.path.abspath(os.getcwd()), 'materials')

        material_files = [f for f in os.listdir(os.path.join(self.path,
                                                             material_path)) if f.endswith('.rad')]
```

It's not an elegant solution, but it works.  Thoughts?

**Comment by mikofski** [comment]
Hi Jason, Thanks for the suggestion. PR #133 was already merged into the dev branch to resolve this issue.

**Comment by cdeline** [comment]
Fixed in 0.3.3 release
